### PR TITLE
Docker Dev-Setup 

### PR DESCRIPTION
**Pull request by dnlkoch**
This suggests to start the webpack development server inside a docker container to be included in the given project SHOGun docker setup. 

Steps to apply this setup:

- `docker-compose.yml`:

```yml
shogun-admin:
  container_name: shogun-admin
  build:
    context: ../../shogun-admin
    dockerfile: Dockerfile.dev
  ports:
    - 9090:9090
  volumes:
    - ../../shogun-admin:/app
```

- nginx config:

```nginx
location /admin/ {
  proxy_pass https://shogun-admin:9090/;
}
```

Afterwards just start the compose setup as usual (e.g. via `docker-compose up`).

Please check @KaiVolland.

### CODE REVIEW

File: webpack.common.config.js [hunk 1]
```diff
@@ -94,9 +94,6 @@ module.exports = {
         from: './assets/fallbackConfig.js',
         to: 'fallbackConfig.js'
       }],
-    }),
-    new SimpleProgressWebpackPlugin({
```
Comment (KaiVolland) [inline webpack.common.config.js:98]: Please remove the import too if not used anymore. Also consider to remove it from the package.json if its not used in other environements.

File: nginx/templates/default.conf.template [hunk 2]
```diff
@@ -1,19 +0,0 @@
-server {
```
Comment (KaiVolland) [inline nginx/templates/default.conf.template:1]: This file is still used in the Dockerfile (not Dockerfile.dev). So it should not be removed.

**Review by KaiVolland — CHANGES_REQUESTED** [review]

File: webpack.common.config.js (hunk 1 above)
Comment (dnlkoch) [reply webpack.common.config.js:98]: Dependency removed in 4e4fe60a35013b904566cf6cde1a58945a7479eb.

File: nginx/templates/default.conf.template (hunk 2 above)
Comment (dnlkoch) [reply nginx/templates/default.conf.template]: Right, readded in b3a7afb0e476e70474146a71fff346ac22e7418b.

**Review by hwbllmnn — APPROVED** [review]

**Review by KaiVolland — APPROVED** [review]

**Comment by dnlkoch** [comment]
Thanks for the reviews!

**Pull request merged**